cmd/internal/utils/files.go

```diff
@@ -80,3 +80,24 @@ func IsCommandPresent(command string) bool {
 
 	return true
 }
+
+func CopyDirectory(osfs afero.Fs, dir string, dest string) error {
+
+	contents, err := os.ReadDir(dir)
+	if err != nil {
+		return err
+	}
+
+	for _, content := range contents {
+		if content.Type().IsDir() {
+			err = CopyDirectory(osfs, dir+"/"+content.Name(), dest+"/"+content.Name())
```

Comment:
This could be dangerous when there is a symlink so this can nest very deep. @majst01 probably has some suggestions how to do this better.

Reply:
Lets wait for https://github.com/golang/go/issues/62484